Library/Formula/awf.rb

```diff
@@ -0,0 +1,30 @@
+class Awf < Formula
+  desc "awf (A Widget Factory) is a theme preview application for gtk2 and gtk3"
+  homepage "https://github.com/valr/awf"
+  url "https://github.com/valr/awf/archive/v1.3.0.tar.gz"
+  sha256 "911ded8a307beecbb8ee1dab489fa7b1aa9d7965cb34bd938482220505814fa4"
+  head "https://github.com/valr/awf.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gtk+"
+  depends_on "gtk+3"
+
+  def install
+    inreplace "src/awf.c", "/usr/share", "#{HOMEBREW_PREFIX}/share"
```

Comment:
I don't see how that would work. Are you saying that /usr/local/share gets a matching symlink in every keg ? For me, I only get a link *from* the higher brew prefix *to* the corresponding keg.
Which means that if I want to find the global themes, I need to look in the global directory rather than the datadir in my own little installation where no-one else will install any themes...

Incidentally, this is the same bug as in gtk+ (#21412) and why it only finds "Raleigh" and friends.
Hardcoding to "/usr/share/theme" is lame anyway, better would be to use `gtk_rc_get_theme_dir`. Except in homebrew, where that is broken and needs environment variables to be set in order to work. But normally, one wants to avoid hardcoding paths to prefixes (relocatable software).